tests/langchain/test_langchain_model_export.py

```diff
@@ -61,6 +61,19 @@
 
 from tests.helper_functions import pyfunc_serve_and_score_model
 
+try:
+    import langchain_community
+
+    # this kwarg was added in langchain_community 0.0.27, and
+    # prevents the use of pickled objects if not provided.
+    VECTORSTORE_KWARGS = kwargs = (
```

Comment:
```suggestion
    VECTORSTORE_KWARGS = (
```

Reply:
oops i must have autocompleted something, thanks for the catch!

Reply:
Lol this parameter looks scary `allow_dangerous_deserialization`

Reply:
But it's just the test, we should be fine